test/fakeintake/server/server.go

```diff
@@ -152,6 +154,17 @@ func WithPort(port int) Option {
 	return WithAddress(fmt.Sprintf("0.0.0.0:%d", port))
 }
 
+// WithStoreDriver changes the store driver used by the server
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
I am wondering why we added `WithAddress`, seems like it it never used. Why should we change the listening address?

Reply:
I don't think there is any use-case. Perhaps we should consider removing it in a future iteration

Reply:
I added it because if you bind to `0.0.0.0` in unit tests then when running on MacOS you'll get very annoying pop-ups 🙃 
The only place it's used is in unit test to set `127.0.0.1` instead

Reply:
If we always bind to `127.0.0.1`, it would still listen when running on a container, wouldn't it?